arch/xtensa/src/common/xtensa_int_handlers.S

```diff
@@ -768,13 +756,13 @@ _xtensa_level6_handler:
 
 	/* Restore only level-specific regs (the rest were already restored) */
 
-	l32i	a0, a2, (4 * REG_PS)			/* Retrieve interruptee's PS */
+	l32i	a0, a2, (4 * REG_PS)      /* Retrieve interruptee's PS */
 	wsr		a0, EPS_6
-	l32i	a0, a2, (4 * REG_PC)			/* Retrieve interruptee's PC */
+	l32i	a0, a2, (4 * REG_PC)      /* Retrieve interruptee's PC */
 	wsr		a0, EPC_6
-	l32i	a0, a2, (4 * REG_A0)			/* Retrieve interruptee's A0 */
-	l32i	sp, a2, (4 * REG_A1)			/* Remove interrupt stack frame */
-	l32i	a2, a2, (4 * REG_A2)			/* Retrieve interruptee's A2 */
+	l32i	a0, a2, (4 * REG_A0)      /* Retrieve interruptee's A0 */
+	l32i	sp, a2, (4 * REG_A1)      /* Retrieve interrupt stack frame */
+	l32i	a2, a2, (4 * REG_A2)      /* Retrieve interruptee's A2 */
```

Comment:
```suggestion
	l32i	a0, a2, (4 * REG_A0)			/* Retrieve interruptee's A0 */
	l32i	sp, a2, (4 * REG_A1)			/* Retrieve interrupt stack frame */
	l32i	a2, a2, (4 * REG_A2)			/* Retrieve interruptee's A2 */
```

Reply:
Why?

Reply:
This is just to aline with code `call0 _xtensa_context_restore /* (preserves a2) */` above as it was initially. But let's keep spaces and migrate to spaces globally for the whole project.

Reply:
Well, in my editor they are already aligned (this is the issue with tabs).

Reply:
It depends how many spaces are configured to display with a single TAB. You can try to play with that setting to see how formatting changes. Probably in your editor it is 2 spaces per TAB

Reply:
Yes it is 2 spaces. This is how many spaces we've been using for tabs, whether for Makefiles or assembly files. Github doesn't necessary follow that and this is why it looks different.

Reply:
But, the assemble file under arm assume 8 space per tab, I think.

Reply:
No. The issue is that other lines use tabs for alignment, but you added 2 spaces per per TAB  on this change and GitHub shows that. If someone will open the file with editor that has 4 space per TAB view (for example), then  formatting will be lost. I just propose to use TABs in this file (actually either TABs or spaces, but not a mix) so view will be agnostic to editor settings.

Reply:
Yes, I agree that the tab/space usage should be consistent, either tab or space is fine, but not mix. Since many assemble files under other arch use tab, if you decide switch to space, please change all of them, thanks.